taskcluster/fenix_taskgraph/release_promotion.py

```diff
@@ -0,0 +1,150 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from __future__ import absolute_import, print_function, unicode_literals
+
+from taskgraph.actions.registry import register_callback_action
+
+from taskgraph.util.taskcluster import get_artifact
+from taskgraph.taskgraph import TaskGraph
+from taskgraph.decision import taskgraph_decision
+from taskgraph.parameters import Parameters
+from taskgraph.util.taskgraph import find_decision_task, find_existing_tasks_from_previous_kinds
+
+from .parameters import resolve_release_type
+
+RELEASE_PROMOTION_PROJECTS = (
+    "https://github.com/JohanLorenzo/fenix",
```

Comment:
Do we want to update this list before landing? I'm ok with a followup once we're ready if this was intentional.

Reply:
I'd like to enable the production repo in a followup. There are still a couple of issues to fix (like the mark-as-ship task) before we're ready.